backend/src/test/java/ai/verta/modeldb/blobs/DiffAndMerge.java

```diff
@@ -20,11 +20,11 @@ public void diffAndMerge(AutogenBlob a, AutogenBlob b) throws ModelDBException {
     AutogenBlobDiff d = DiffComputer.computeBlobDiff(newA, newB);
 
     // Applying the diff on top of the original A should get original B
-    AutogenBlob diffedB = DiffMerger.mergeBlob(newA, d);
+    AutogenBlob diffedB = DiffMerger.mergeBlob(newA, d, new HashSet<String>());
```

Comment:
I think this PR is already created for these tests fixes: https://github.com/VertaAI/modeldb/pull/484 @ravishetye

Reply:
reverted